eth/stagedsync/stage_headers.go

```diff
@@ -172,14 +177,14 @@ func HeadersPOS(
 		return err
 	}
 	if parent != nil && parent.Hash() == header.ParentHash {
-		if err := cfg.hd.VerifyHeader(&header); err != nil {
+		/*if err := cfg.hd.VerifyHeader(header); err != nil {
```

Comment:
What was the problem with header verification?

Reply:
GasLimit was wrongly detected as faulty, (aka too much change), i am not sure if this is a problem regarding the engine or regarding mergemock(this only happens when running mergemock), but until we connect to kintsugi and see for ourselves i think this can be commented out

Reply:
We should not to comment out header verification but rather investigate any issues with it and fix them. We can do that later, and for now I don't see a point in merging disabling header verification. Do you have instructions on how to test Erigon with mergemock handy? (So that I could reproduce the gas limit issue.)

Reply:
I will write some notes then on the procedure and send them to you... i will uncomment this in the meanwhile...